test/queueTest.ts

```diff
@@ -168,6 +168,7 @@ describe('Test Queue class', function() {
   });
 
   it('should getReply as a stream', async function() {
+    if (process.env.SKIP_STREAM) return;
```

Comment:
nit: https://stackoverflow.com/a/42586302/1138559
It seems more elegant, haven't tried it though